deps.ts

```diff
@@ -50,12 +50,8 @@ export type { Cookie } from "https://deno.land/std@0.92.0/http/cookie.ts";
 
 export { green, red } from "https://deno.land/std@0.92.0/fmt/colors.ts";
 
-export {
-  IndexService,
-} from "https://raw.githubusercontent.com/drashland/services/v0.2.1/index/index_service.ts";
-export type {
-  ISearchResult,
-} from "https://raw.githubusercontent.com/drashland/services/v0.2.1/index/index_service.ts";
+export { Moogle } from "https://deno.land/x/moogle@v1.0.0/mod.ts";
+
 // export {
 //   IndexService,
 //   ISearchResult,
```

Comment:
I'm on the phone, so I can't quite understand, but these are commented lines that probably should be removed.

Reply:
https://github.com/drashland/deno-drash/pull/498/commits/6ed4c759f155262e2eb28e163323e7803fe29061